hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java

```diff
@@ -253,34 +258,36 @@ private void addManCenterInfo(Node hazelcastNode, int clusterSize, PhoneHomePara
             }
             connection.setConnectTimeout(CONNECTION_TIMEOUT_MILLIS);
             connection.setReadTimeout(CONNECTION_TIMEOUT_MILLIS);
+
             // if management center is not running,
             // connection.getInputStream() throws 'java.net.ConnectException: Connection refused'
             inputStream = connection.getInputStream();
-            responseCode = connection.getResponseCode();
             reader = new InputStreamReader(inputStream, "UTF-8");
-            JsonObject mcPhoneHomeInfoJson = JsonValue.readFrom(reader).asObject();
-            version = JsonUtil.getString(mcPhoneHomeInfoJson, "mcVersion");
-            license = JsonUtil.getString(mcPhoneHomeInfoJson, "mcLicense", null);
+            JsonObject mcPhoneHomeInfoJson = Json.parse(reader).asObject();
```

Comment:
`JsonValue.readFrom(Reader)` was deprecated, `Json.parse(Reader)` was proposed as replacement in its JavaDoc.